testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/AbstractPage.java

```diff
@@ -20,6 +20,7 @@
 import org.jboss.arquillian.test.api.ArquillianResource;
 import org.junit.Assert;
 import org.keycloak.testsuite.util.oauth.OAuthClient;
+import org.keycloak.testsuite.util.webdriver.PageUtils;
```

Comment:
Just leave the PageUtils in the old testsuite, and when migrating use the one from `test-framework/ui` or if it's needing just the title then use it from AbstractPage. However, I doubt there really is many cases that need the title and should rather do it based on the `data-page-id`